go.mod

```diff
@@ -80,13 +79,14 @@ require (
 	golang.org/x/net v0.25.0
 	golang.org/x/sys v0.20.0
 	google.golang.org/grpc v1.63.2
-	google.golang.org/protobuf v1.34.0
+	google.golang.org/protobuf v1.34.1
```

Comment:
Why did you get all these upgrades? We should not mix them in this pr. Only run go mod tidy.